eventbus_test.go

```diff
@@ -0,0 +1 @@
+package eventbus_test
```

Comment:
This this empty. Let's add real test or include it in the next PR.

Reply:
Let's keep this PR short, so in the next PR will add the tests